tests/test_artifact_removal.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+import pytest
+from playwright.async_api import async_playwright
+
+example_data = [
+    {
+        "file_name": "test_artifact_page.mhtml",
+        "artifact_selectors": [
+            "#__tarsier_id",
+        ],  # TODO: add more selectors once colour tagging is merged
+    },
+]
+
+tarsier_functions = [
+    (
+        "page_to_text",
+        {"tag_text_elements": True, "keep_tags_showing": True},
+        ["remove_tags"],
```

Comment:
to make this more concrete and avoid dealing with strings, you can make these lambda functions that take in tarsier/the page as a param, and then you can call the functions with the desired params directly

Reply:
also avoids this get attribute stuff